broker/consumer/vm_startup.sh

```diff
@@ -87,17 +87,28 @@ do
     # which kills the while loop. no working suggestions found.
     # passing the error with `|| true`
 
+    
+    IFS=', ' read -r -a KAFKA_TOPIC_ARRAY <<< "$KAFKA_TOPIC"
+    KAFKA_TOPIC_PRESENT=()
+
     # check if our topic is in the list
```

Comment:
Update this comment to better describe what will happen in this code block. Include info about different cases, e.g., multiple topics but one goes live before the other. You may want to point at issue #212 where the  ongoing discussion will occur and options & unknowns can be described in more detail.

Reply:
I added a more descriptive comment, please let me know what you think. I felt it might be more appropriate to keep the discussion about the different cases in Issue #212, but if you disagree, let me know and I will update the comment again